cli/crates/server/src/udf_builder.rs

```diff
@@ -18,20 +18,30 @@ async fn run_command<P: AsRef<Path>>(
     environment: &[(&'static str, &str)],
 ) -> Result<(), JavascriptPackageManagerComamndError> {
     let command_string = format!("{command_type} {}", arguments.iter().format(" "));
-
-    trace!("running '{command_string}'");
+    let current_directory = current_directory.as_ref();
+    if !current_directory.exists() {
+        return Err(JavascriptPackageManagerComamndError::NotFound(
+            None,
+            format!("Working directory not found: {current_directory:?}"),
+        ));
```

Comment:
Following on from my previous comment: should this be another error variant, rather than re-using `NotFound`?

`Working directory not found: {}` seems like a better error message than `Could not find None: Working directory not found: {}`

Reply:
agreed, fixing